po/fi.po

```diff
@@ -720,9 +721,8 @@ msgstr "Aikaväli"
 
 #: gnucash/gnome/assistant-acct-period.c:590
 #: gnucash/gtkbuilder/dialog-book-close.glade:86
-#, fuzzy
 msgid "Closing Date"
-msgstr "Vientipäivä"
```

Comment:
Why simply delete this, or was it a mistake? If you think it's wrong, could you suggest a better translation?

Reply:
In some cases I thought there was misleading translation. Having untranslated string is better than having misleading translation. In this case "Vientipäivä" would mean export date.

You review is appreciated. I am not sure I understood this right. What I understood was that this would be the last day of the books. It could be "Tilinpäätöspäivä". I just was not sure at the time I looked at it.

Reply:
> In some cases I thought there was misleading translation. Having untranslated string is better than having misleading translation. 

In such a case you can insert the `, fuzzy`flag. Then the translation is not used.

Reply:
Thanks. I'll fuzzy them if similar comes up. I changed this to `Tilinpäätöspäivä`. I see it being used meaning the date you have to value your assets (stock prices etc.) when closing books, which I understood this field to mean.

I did remove quite a few other translations that were already fuzzy as I thought they were created by msgmerge and too far from correct to be helpful. I rather view them as untranslated when I get time to think for a correct translation.